src/libraries/pkg/test/project.csproj.template

```diff
@@ -7,13 +7,27 @@
     <TestPackageVersion>{PackageVersion}</TestPackageVersion>
     <!-- Turn off end of life target framework checks as we intentionally build older .NETCoreApp configurations. -->
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <NETCoreAppMaximumVersion>99.0</NETCoreAppMaximumVersion>
```

Comment:
Nit, you could move this to: https://github.com/dotnet/runtime/blob/master/src/libraries/pkg/test/props/Directory.Build.props

Reply:
This property needs to be set in .csproj or in .targets otherwise it will be overwritten by the sdk.